### DISCUSSION TITLE
Is anyone else being banned a lot faster lately?

### DISCUSSION THREAD

**Discussion by diekunstderfuge** [danielkrupinski/Osiris#3815] [General]
Lately I have been banned a lot fast than I used to be, even though I'm not doing nothing different than before, so it must be a recent thing. One account was banned with only 4 hours of playing time.

I used to cheat blatantly for a lot longer in the past, like more than 100 hours without being banned.

Is it happening with anyone else?

**Comment by CertifiedClown** [comment]
no. it depends on what your using! have been using osiris legit on brand new account for past 3 weeks and still not banned

**Reply by diekunstderfuge** [reply]
So, the thing is that I haven't changed anything that I used to do, but recently I have been banned very quickly. So I think there must be some other factor...

**Reply by CertifiedClown** [reply]
> So, the thing is that I haven't changed anything that I used to do, but recently I have been banned very quickly. So I think there must be some other factor...

Im only using ( backtrack - 200 and Inventory changer ) and still not banned. You can try that. also dont recommend using any type of chams or esp from my experience you will always get banned 100%

**Reply by CertifiedClown** [reply]
I can send you my config if you want to just gmail me 1310s@proton.me
